src/manager-iterator/Chart.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v2
+name: manager-iterator
+description: A Helm chart for Kubernetes
```

Comment:
this should be added in harness-core, this repo is no longer actively operational